Automattic/wp-calypso
# Store Services: Add a "Use address as entered" when an verified address is being edited

#### Changes proposed in this Pull Request

This PR addresses some issues in the shipping label dialog:

* Adds a "Use address as entered" button while editing addresses in order to allow their usage without preliminary verification or when verification failed.
* Modifies the behaviour of the "Verify Address" button to always trigger a verification, even if the address has been normalized.

#### Testing instructions

1. Open the dialog to create a new label
2. Enter an address that is valid but cannot be verified
3. Click `Verify Address` to enter the `We were unable to automatically verify the address` state.
4. Click on `Edit Address`. *On the new screen, the "Verify Address" button will not be active before the introduction of new changes, but the new "Use address as entered" button will be active and usable.* Click the new button.
5. You can go back to the previous step (Destination/Origin) and the "Verify Address" button will actually verify the address, instead of just proceeding to the next step.

[![](https://cdn-std.dprcdn.net/files/acc_693496/IHLW2L)](https://cld.wthms.co/IHLW2L)

Fixes https://github.com/Automattic/woocommerce-services/issues/1492
